src/components/_global/BalCard/BalCard.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <div :class="['bal-card overflow-hidden', cardClasses]">
+  <div :class="['bal-card', cardClasses]">
+    <div v-if="imgSrc" class="feature" :style="featureStyles" />
```

Comment:
We can handle feature images directly in BalCard, it's not that complicated a component and makes sense that it should be able to handle a few variations of card types.